### ISSUE TITLE
Long hebrew abstract breaks compilation

### ISSUE THREAD

**Issue by avivrosenberg** [eyalroz/technion-iit-thesis#52]
First, much gratitude for your efforts maintaining this template.

I'm working on a thesis using the template. Obviously I left the Hebrew abstract part for last, and suddenly I noticed that the document compiling breaks once my Hebrew abstract is longer than one page.

Initially I though that it must be something wrong with my specific thesis code (I did some hacks to the template). But after hours of trying to remove all my changes to find out what what's breaking it, I kept getting the same error.

So I decided to check out `master` on this repo and try with a fresh template.
It seems that on `master`, just running this command from the readme (with a cleanup first):
```shell
latexmk -C && latexmk -xelatex thesis
```
Immediately produces the error I got in my thesis, which is this:
```shell
! Extra }, or forgotten \endgroup.
l.69 \egroup
```
Then I checked to see whether on the clean template from `master` it's also related to the length of the Hebrew abstract. And indeed, if I go to `front/abstract.tex` and comment out the `\newpage` line and the line below it (within the `\abstractHebrew{}`) block, then the template compiles perfectly with the command above.

I'm using a local TexLive 2021 on a mac (via mactex). I later tried on Overleaf with legacy TexLive 2018 and it does seem to resolve it. However it would really be much better to be able to use TexLive 2021 because with 2018 I have other weird issues with my thesis and prefer not to resort to Overleaf. @eyalroz any idea how it can solve this or even just work around it? Thanks.

**Comment by eyalroz** [comment]
This seems to be a duplicate of #33 and is not really about the length of the Hebrew extended-abstract. Please have a look and try the suggestion there.

**Comment by avivrosenberg** [comment]
Thanks, it was indeed the same error. I guess the page order issue only triggered when the hebrew abstract was longer than one page, since it did compile fine before that. Anyway, I upgraded to TL2022 but it still only had polyglossia `1.53` by default, while the fix should be in `1.54`. After also updating polyglossia to `1.60` I can confirm the issue is resolved. Thanks for your help!
